src/editor/api/auth.js

```diff
@@ -1,18 +1,22 @@
 import { GoogleAuthProvider, signInWithPopup } from 'firebase/auth';
 import { auth } from '../services/firebase';
 import { sendMetric } from '../services/ga';
+import posthog from 'posthog-js';
 
 const signIn = async () => {
   try {
-    const {
-      user: {
-        metadata: { creationTime, lastSignInTime }
-      }
-    } = await signInWithPopup(auth, new GoogleAuthProvider());
-
+    const { user } = await signInWithPopup(auth, new GoogleAuthProvider());
+    posthog.identify(user.uid, {
+      email: user.email,
+      name: user.displayName
+    });
     // first signIn to ga
-    if (creationTime !== lastSignInTime) return;
+    if (user.metadata.creationTime !== user.metadata.lastSignInTime) return;
     sendMetric('Auth', 'newAccountCreation');
+    posthog.capture('user_signed_up', {
+      email: user.email,
+      name: user.displayName
+    });
```

Comment:
If you do business in EU, you will need to have a proper Privacy Policy document that says you send email and the fullname of the user to a third party service.

Reply:
https://github.com/3DStreet/3dstreet/issues/664